client/offchain/src/api.rs

```diff
@@ -180,6 +197,48 @@ impl<Storage: OffchainStorage> OffchainExt for Api<Storage> {
 	) -> Result<usize, HttpError> {
 		self.http.response_read_body(request_id, buffer, deadline)
 	}
+
+	fn pollable_wait(
```

Comment:
I think we might consider placing async methods in a separate - for instance `offchain::async::wait` & `offchain::async::timer`. Long-term we will either support both (sync & async), but hopefully be able to implement the `sync` stuff, simply as `block_on(async_stuff)` or we deprecate `sync` completely when we are confident enough it works and the ergonomics is superior.

Reply:
As a side note, I did think about it at first but I was concerned with ergonomics - `async` is a keyword and needs a raw identifier treatment, e.g. `offchain::r#async::wait`. Is that something that we might consider as acceptable or should we think of a different word/term?